doc/Command_Index_User.tex

```diff
@@ -565,28 +568,40 @@ \subsubsection{Glyph Alteration}
   \#5 & dimension & vertical shift of the bar glyph.\\
 \end{argtable}
 
+\macroname{\textbackslash greseperator}{\{\#1\}\{\#2\}}{gregoriotex-symbols.tex}
```

Comment:
Misspelled; should be greseparator (the spelling is correct in the description below).